torch/_functorch/_aot_autograd/traced_function_transforms.py

```diff
@@ -740,9 +749,19 @@ def fw_fn(*primals):
     def metadata_fn(*primals):
         return inner_fn(fw_only, primals, use_trace_joint=False)
 
-    args_unwrapped = unwrap_tensor_subclasses(
-        args, is_joint_structure=is_joint_structure
-    )
+    if is_joint_structure:
+        args_unwrapped = (
+            unwrap_tensor_subclasses(
+                args[0], is_runtime=False, append_symints=True, subclass_metas=None
+            ),
+            unwrap_tensor_subclasses(
+                args[1], is_runtime=False, append_symints=False, subclass_metas=None
```

Comment:
e.g. here: some explicit comment explaining why we append_symints for primals but not tangents would be great (the reason being that any symints needed in the backward will be automatically saved by the partitioner, and we won't need an explicit wrapper in the backward at runtime that passes in these extra inputs)